README.md

```diff
@@ -73,17 +73,11 @@ $ cargo build --release
 To connect NEAR Indexer for Explorer to the specific chain you need to have necessary configs, you can generate it as follows:
 
 ```bash
-$ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
+$ ./target/release/indexer-explorer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
 ```
 
 The above code will download the official genesis config and generate necessary configs. You can replace `testnet` in the command above to different network ID (`betanet`, `mainnet`).
 
-**NB!** According to changes in `nearcore` config generation we don't fill all the necessary fields in the config file.
```

Comment:
While I agree with removing this paragraph, I would like to keep the links to configs as they might be useful for future reference.

Reply:
Ok, but what is the way to keep them? Just keep the links or like "Here are the links to initial configs"?

Reply:
Yes, please, phrase it as these are the default config files that one could use just for the reference.